.github/workflows/release-alpha.yml

```diff
@@ -1,5 +1,9 @@
 name: Release alpha version
-on: [workflow_dispatch]
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Using the `push` trigger seems fine for me. 

However with this change we are going to trigger 2 separate and identical builds on the main branch, one that comes from the `build` workflow and one from this `release-alpha` workflow. So my question is if we should get rid of the `release-alpha` workflow, and instead do this in the `build` workflow in a separate job that only runs if the branch is `main`